src/components/SignupPopup.vue

```diff
@@ -0,0 +1,34 @@
+<script setup>
+import github from '@/assets/svgs/github.svg'
+</script>
+
+<template>
+  <!-- make display flex and remove hidden -->
```

Comment:
remove comment

Reply:
I need them for develop that element later